mappings/net/minecraft/client/render/entity/feature/ArmorFeatureRenderer.mapping

```diff
@@ -2,6 +2,22 @@ CLASS net/minecraft/class_970 net/minecraft/client/render/entity/feature/ArmorFe
 	FIELD field_4829 ARMOR_TEXTURE_CACHE Ljava/util/Map;
 	FIELD field_4830 modelLeggings Lnet/minecraft/class_572;
 	FIELD field_4831 modelBody Lnet/minecraft/class_572;
+	METHOD method_23192 renderArmorParts (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;ILnet/minecraft/class_1738;ZLnet/minecraft/class_572;ZFFFLunresolved_class;)V
+		ARG 1 matrixStack
+		ARG 2 layeredVertexConsumerStorage
+		ARG 4 armorItem
+		ARG 6 bipedEntityModel
+		ARG 7 lowerParts
+		ARG 8 r
+		ARG 9 g
+		ARG 10 b
+		ARG 11 textureSuffix
 	METHOD method_4169 renderArmor (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;Lnet/minecraft/class_1309;FFFFFFFLnet/minecraft/class_1304;I)V
+	METHOD method_4170 setVisible (Lnet/minecraft/class_572;Lnet/minecraft/class_1304;)V
 	METHOD method_4172 getArmor (Lnet/minecraft/class_1304;)Lnet/minecraft/class_572;
 	METHOD method_4173 isLegs (Lnet/minecraft/class_1304;)Z
+	METHOD method_4174 getArmorTexture (Lnet/minecraft/class_1738;ZLunresolved_class;)Lnet/minecraft/class_2960;
```

Comment:
`Lunresolved_class;` :thinking:

Reply:
Looks like Knit messed some stuff up. I fixed it manually with diff and Enigma.